src/usdb_syncer/gui/settings_dialog.py

```diff
@@ -24,6 +24,8 @@ def _populate_comboboxes(self) -> None:
             self.comboBox_encoding.addItem(str(encoding), encoding)
         for newline in settings.Newline:
             self.comboBox_line_endings.addItem(str(newline), newline)
+        for apostrophe in settings.Apostrophe:
+            self.comboBox_apostrophes.addItem(str(apostrophe), apostrophe)
```

Comment:
This has apparently been the final straw for `pylint` to rate the `_populate_comboboxes` function as "too complex".
I personally don't think this is a useful measure and should be deactivated. Alternatively, one could put the combobox names and setting enums into lists and have an outer for loop that iterates over these, but this isn't much prettier imo